### PR TITLE
fix: Chaindesk retriever

### PR DESCRIPTION
**Pull request by gmpetrov**

### CODE REVIEW

File: libs/langchain/langchain/retrievers/chaindesk.py [hunk 1]
```diff
@@ -13,17 +13,17 @@
 class ChaindeskRetriever(BaseRetriever):
     """`Chaindesk API` retriever."""
 
-    datastore_url: str
+    datastore_id: str
```
Comment (eyurtsev) [inline libs/langchain/langchain/retrievers/chaindesk.py:16]: @gmpetrov thanks for the fix. The is a breaking change for the API. Could you explain why this change is required (why is most of the URL hard-coded in the code?) and perhaps find a way to do it in a backwards comaptible way?

**Comment by baskaryan** [comment]
Apologies for the slow review! Pr has some merge conflicts, happy to re-review if you'd like to resolve
